confidence/models.py

```diff
@@ -46,17 +47,21 @@ def unwrap(source: typing.Any) -> typing.Any:
     return source
 
 
-def merge(*sources: typing.Mapping[str, typing.Any], missing: typing.Any = None) -> 'Configuration':
+def merge(*sources: typing.Mapping[str, typing.Any],
+          missing: typing.Any = None,
+          secrets: typing.Any = None) -> 'Configuration':
```

Comment:
Mypy fails to infer the set being constructed below correctly when this is set to `typing.Optional[Secrets]`, meh :thinking: